nuitka/distutils/Build.py

```diff
@@ -31,6 +31,9 @@ def suppress_known_deprecation():
 else:
     suppress_known_deprecation = setuptools.build_meta.suppress_known_deprecation
 
+SETUPTOOLS_ENABLE_FEATURES = os.getenv("SETUPTOOLS_ENABLE_FEATURES", "").lower()
+LEGACY_EDITABLE = "legacy-editable" in SETUPTOOLS_ENABLE_FEATURES.replace("_", "-")
```

Comment:
I was wondering if this duplicates code from setuptools, but it's not too important I think.

Reply:
This is for full compatibility with the `setuptools` configuration, if the user runs with `SETUPTOOLS_ENABLE_FEATURES="legacy-editable"` it may cause `_BACKEND.get_requires_for_build_editable`, `_BACKEND .prepare_metadata_for_build_editable` and `_BACKEND.build_editable` undefined because they are all declared in the `setuptools.build_meta` code under the `if not LEGACY_EDITABLE` condition. Also, since `LEGACY_EDITABLE` is not exported in `setuptools.build_meta`, the only way to get the truth of `LEGACY_EDITABLE` is to add the same two lines as in the `setuptools.build_meta` code here, So I wrote it like this. Or do you have a better way to do it?

Reply:
Well, not being in `__all__` doesn't prevent usage. So, e.g. the current code does this:

```
if not hasattr(setuptools.build_meta, "suppress_known_deprecation"):
```

I believe that's also for working with older setuptools. In that way, older or newer setuptools decides stuff, and not us. 

So `LEGACY_EDITABLE = getattr(setuptools.build_meta, "LEGACY_EDITABLE")` would probably be more future proof. I will just change it on develop after the hotfix release for 1.8 to do it that way. Most important is that it works more often than before for the 1.7 release.

Reply:
Like I said, no big thing. But in case they somehow change their mind, it's a good idea. And now e.g. Nuitka would attempt to use it with a setuptools that doesn't do it, but that's going to be rarely possible I guess, but like I said, just going to add that, and ask you to retest, hold on.

Reply:
Okay, and I noticed that the commit for this PR that you merged yesterday is missing from the develop branch, so do I need to resubmit a new PR?

Reply:
It's on its way into 1.7.8, right now being in hotfix/1.7.8 branch, which I intend to release shortly as well.

Reply:
That's great, I'm waiting for this release.🥳